src/routes/routes.ts

```diff
@@ -28,7 +28,8 @@ import * as wrongStatementOfCapitalRoute from "../controllers/incorrect-informat
 import * as wrongShareholdersRoute from "../controllers/incorrect-information/wrong.shareholders.controller";
 import * as wrongRegisteredOfficeAddressRoute from "../controllers/incorrect-information/wrong.registered.office.address.controller";
 import * as wrongRegisterLocationsRoute from "../controllers/incorrect-information/wrong.registers.controller";
-import * as wrongDetailsRoute from "../controllers/incorrect-information/wrong.details.controller";
+import * as wrongDetailsRoute from "../controllers/incorrect-information/wrong.officer.details.controller";
```

Comment:
Could you import this with the name 'wrongOfficerDetailsRoute' which makes clear what it is? Consistent then with 'wrongPscDetailsRoute'.